gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/management/impl/InternalManagementService.java

```diff
@@ -408,7 +411,20 @@ private void handleTableCreate(Object[] data) {
     this.tableMBeanDataUpdater.addUpdatable(tableMBeanBridge.getFullName(), tableMBeanBridge); // start updates after creation
   }  
 
-  List<String> getTableDefinition (String parentSchema, String tableName){   
+  private boolean hasInternalTableSchema(GemFireContainer container) {
+    for (String schema : CallbackFactoryProvider.getStoreCallbacks().getInternalTableSchema()) {
+      if (schema.equalsIgnoreCase(container.getSchemaName())) {
+        return true;
+      }
+    }
+    // This constant is defined at snappy-store side, so won't be included in above response from StoreCallback.
+    if (StoreCallbacks.INTERNAL_SCHEMA_NAME.equalsIgnoreCase(container.getSchemaName())) {
+      return true;
```

Comment:
Also do we really want to exclude StoreCallbacks.INTERNAL_SCHEMA_NAME? That can be useful to show real column table memory. Can be handled in future (pulse already out of 0.1preview).